### ISSUE TITLE
Maybe add image of isotope pattern in 1D

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/Tutorials#82]
![50c_iso](https://user-images.githubusercontent.com/5803621/31342468-27383ffe-ad0d-11e7-9b62-ad2cc507508f.png)

**Comment by hroest** [comment]
:+1: but what about annotating  the other peaks, e.g. just for clarity ? Also I dont quite understand what 12C50 means, e.g. is it 50 12C atoms?

**Comment by timosachsenberg** [comment]
yes its 50 carbon atoms. Its not ideal (aka theoretical isotope pattern) but I currently don't have a better one available

**Comment by hroest** [comment]
I think its good, but it would be helpful to have the axis zoomed in a bit or at least label 601, 602 etc so that it is clear what the other peaks are. Also it could be helpful to label the other peaks 13C1 49C12  and 13C2 48C12 for clarity...
